### PR TITLE
fix:  change password on dashboard

### PR DESCRIPTION
**Pull request by iresharma**
## Related issues
https://github.com/supertokens/supertokens-python/issues/312

## Test Plan
- Tried changing password for multiple users to invoke the API
- Passed all existing tests

## Checklist for important updates

-   [x] Changelog has been updated
-   [ ] `coreDriverInterfaceSupported.json` file has been updated (if needed)
    -   Along with the associated array in `supertokens_python/constants.py`
-   [ ] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [x] Changes to the version if needed
    -   In `setup.py`
    -   In `supertokens_python/constants.py`
-   [x] Had installed and ran the pre-commit hook
-   [x] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [ ] If have added a new web framework, update the `supertokens_python/utils.py` file to include that in the `FRAMEWORKS` variable
-   [ ] If added a new recipe that has a User type with extra info, then be sure to change the User type in supertokens_python/types.py
 
## Remaining TODOs for this PR

-   [ ] Item1
-   [ ] Item2

### CODE REVIEW

**Review by iresharma — COMMENTED** [review]
Self Review done

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## unreleased
 
+## [0.12.7] - 2023-04-19
```
Comment (rishabhpoddar) [inline CHANGELOG.md:10]:
```suggestion
## [0.12.8] - 2023-04-19
```

File: setup.py [hunk 2]
```diff
@@ -70,7 +70,7 @@
 
 setup(
     name="supertokens_python",
-    version="0.12.6",
+    version="0.12.7",
```
Comment (rishabhpoddar) [inline setup.py:73]:
```suggestion
    version="0.12.8",
```

File: supertokens_python/constants.py [hunk 3]
```diff
@@ -25,7 +25,7 @@
     "2.19",
     "2.20",
 ]
-VERSION = "0.12.6"
+VERSION = "0.12.7"
```
Comment (rishabhpoddar) [inline supertokens_python/constants.py:28]:
```suggestion
VERSION = "0.12.8"
```

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**